akka-typed/src/main/scala/akka/typed/javadsl/Actor.scala

```diff
@@ -55,7 +55,7 @@ object Actor {
    * @see [[Actor#mutable]]
    */
   abstract class MutableBehavior[T] extends ExtensibleBehavior[T] {
-    private val receive: Receive[T] = createReceive
+    private lazy val receive: Receive[T] = createReceive
```

Comment:
I commented on the commit, don't know if that was lost. Due to the concurrency/synchronization constructs that lazy adds we should instead use our own lazy init.

```scala
private var _receive: OptionVal[Receive[T]]
private def receive: Receive[T] = _receive match {
  case OptionVal.None =>
    _receive = createReceive
    _receive
  case OptionVal.Some(r) => r
}
```